bundle.yaml.j2

```diff
@@ -72,7 +72,8 @@ applications:
   {%- endif %}
 
 relations:
-- [prometheus:ingress, traefik]
+- [traefik, prometheus:ingress]
+- [traefik, loki:ingress]
```

Comment:
I would make explicit both sides of the relation just  as the other relations we have in the file